salt_stack/salt/open-files/files/increase_open_files.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+echo -e "fs.file-max=500000" | sudo tee --append /etc/sysctl.conf
```

Comment:
May want to have a check to make sure we are not adding a duplicate key / entry.

Reply:
This key wasn't already in the sysctl.conf file of a new VM.  I checked already.  It wouldn't hurt grep out lines with that in it first.